uniffi_docs/src/lib.rs

```diff
@@ -162,15 +171,53 @@ pub fn extract_documentation(path: &Utf8Path) -> Result<Documentation> {
             syn::Item::Enum(item) => {
                 let name = item.ident.to_string();
                 let description = extract_doc_comment(&item.attrs);
+
+                let members = item
+                    .variants
+                    .iter()
+                    .filter_map(|variant| {
+                        extract_doc_comment(&variant.attrs)
+                            .map(|doc_comment| (variant.ident.to_string(), doc_comment))
+                    })
+                    .collect();
+
                 if let Some(description) = description {
-                    structures.insert(name, Structure { description });
+                    structures.insert(
+                        name,
+                        Structure {
+                            description,
+                            members,
```

Comment:
I think that calling `collect` there could improve the performance when `description == None`.

Reply:
Will do.

Reply:
Done